activerecord/lib/active_record/associations.rb

```diff
@@ -1275,7 +1275,8 @@ module ClassMethods
         #   association will use "person_id" as the default <tt>:foreign_key</tt>.
         #
         #   If you are going to modify the association (rather than just read from it), then it is
-        #   a good idea to set the <tt>:inverse_of</tt> option.
+        #   a good idea to set the <tt>:inverse_of</tt> option
+        #   (it will be automatically detected if the association matches the name of the class).
```

Comment:
The current documentation in the inversing section reads `the association will do its best to match itself up with the correct inverse` but I think what you have written is more technically accurate. Something else that could be potentially misleading is that we're not mentioning it only works for certain association types.